docs/frc/1-Basics/9-Drivetrain.md

```diff
@@ -1,3 +1,300 @@
+
+```java
+@Override
+public void update() {
+    switch (state) {
+        case MANUAL:
+            drivetrain.arcadeDrive(speedForward, speedTurn);
+            break;
+    }
+}
+```
+When the drivetrain state is `MANUAL`, we use `DifferentialDrive`'s `arcadeDrive()` to command our drivetrain's movement by passing in the forward and turn speeds. 
+
+### Other Functions
+
+```java
+    @Override
+    public void displayShuffleboard() {
+        SmartDashboard.putString("Drive State", state.name());
+    }
+
+    @Override
+    public void tuningInit() {
+        
+    }
+
+    @Override
+    public void tuningPeriodic() {
+
+    }
+```
+These are functions used mainly setup for robot testing/diagnostics. We will discuss these further in the next few sections. 
+
+As of right now, we have declared and initialized everything, and added the `update()` function that commands the drivetrain's motors based off the current state. However, we have no way of actually changing the state or `speedForward` and `speedTurn` variables yet. We can create a function to address these issues, and might call it `manualDrive()`:
+
+```java
+    public void manualDrive(double speedForward, double speedTurn) {
+        this.speedForward = speedForward;
+        this.speedTurn = speedTurn;
+
+        state = defaultState;
+    }
+```
+With this, we can modify the driving speeds using passed-in values, and then change the state to the default (manual) state. 
+
+With all this done, we're now read to move onto our first drivetrain command!
+
+## Command File
+
+Because the only state we are dealing with is the `MANUAL` state, we'll create a command called `ManualDriveCommand`. 
+
+### Declarations
+```java 
+package frc.robot.commands.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+import java.util.function.DoubleSupplier;
+
+public class ManualDriveCommand extends CommandBase {
+
+    private final Drivetrain drivetrain;
+    private final DoubleSupplier forwardSupplier;
+    private final DoubleSupplier turnSupplier;
+
+    ...
+}
+```
+These are the basic declarations we'll need for this command to work. Just like in the previous examples, we use `DoubleSupplier`s to later store controller joystick inputs, instead of using doubles. 
+
+### Constructor
+
+```java
+    public ManualDriveCommand(Drivetrain drivetrain, DoubleSupplier forwardSupplier, DoubleSupplier turnSupplier) {
+        this.drivetrain = drivetrain;
+        this.forwardSupplier = forwardSupplier;
+        this.turnSupplier = turnSupplier;
+
+        addRequirements(drivetrain);
+    }
+```
+Just like before, we initialize the subsystem object and suppliers with the passed in references. The functions passed in for the suppliers will be functions returning the respective joystick values. Finally, we call `addRequirements()` so that the `CommandScheduler` will know what subsystem is needed for this command. 
```

Comment:
TBH I feel like we can avoid mentioning the `CommandScheduler` specifically since it might confuse people who aren't that familiar with the technical details of how WPILib command based work. We can instead just say so that the command based architecture knows how to handle it.